htlcswitch/switch_test.go

```diff
@@ -1554,6 +1554,11 @@ func TestSwitchAddSamePayment(t *testing.T) {
 	if s.circuits.NumOpen() != 0 {
 		t.Fatal("wrong amount of circuits")
 	}
+	// sleep 16 seconds to give logTicker fwdEventicker and time to tick.
+	// without this the code handling the logTicker and fwdEventicker
+	// may not be called (randomally) during coverage tests and there will
+	// be a drop in coverage stats (build fail)
+	time.Sleep(16 * time.Second)
```

Comment:
We don't really want to introduce more sleeps in the tests...

Reply:
@halseth  Please let me know what you want to do to as an alternative. 
As ugly as this sleep is, without it we can't be sure that these two tickers actually ticks. If the tests run fast (stronger, not loaded machine) the tickers will not tick and the relevant code will not execute. This will cause a failure in coveralls step.

If you want to have a consistent build process, we better have this sleep (or look for something more complicated to ensure the ticker execution).

Let me know what you suggest.

Reply:
Take a look at how other tickers are being mocked: https://github.com/lightningnetwork/lnd/blob/e0baa4969035fc555b185d21acaa0b09aecdef6f/htlcswitch/test_utils.go#L921, might be able to do something similar.

Reply:
Well, I don't think that we want to mock or test these tickers here. The intention is that coveralls produces consistent results. 
Mocking the ticker will prevent that code from running and reduce coverage so what will we gain.

I spend a lot of time on the analysis of the reasons for Travis and coveralls failures. I'm not a maintainer of this project but if I were, I would first make sure there is a stable test/build environment and only after that work on improvements.  
The current situation where a PR may fail on different reasons which have nothing to do with the change done in that PR is not good.

I don't see what is wrong with these 16 seconds sleep.

Reply:
@halseth BTW, see last checks of this simple PR. 2 failures which are unrelated to the change. At least coversall.io is green.

Reply:
If you mock the ticker, you can make it tick immediately, and won't have to wait the 16 seconds on it to happen?

Reply:
@halseth please note that the ticker related change is now on its own PR.